### ISSUE TITLE
Image recognition using Pl@ntNet API

### ISSUE THREAD

**Issue by wgalternate** [MDeLuise/plant-it#332]
### Avoid duplicated feature requests

- [X] There are no open or closed feature requests that are related to this request

### Description

Would it be possible to add a plant image recognition system using the free Pl@ntNet API?
https://my.plantnet.org/

**Comment by MXC48** [comment]
Yes, it would be really great to be able to use Plantnet's free API in the app! That would allow plants to be recognised automatically so that they could be added to the search, for example, and information on how to care for them could be obtained thanks to Flora Codex :)
